lib/report/generateReport.ts

```diff
@@ -224,7 +215,9 @@ export class CoverageView {
 
       this.resultsForRendering = this.coverageResultsForRendering.map((item) => {
         const data = this.validationResultsForRendering.find(
-          (i) => i.specFilePath && item.spec.includes(i.specFilePath)
+          (i) =>
+            i.specFilePath &&
+            item.spec.split(path.win32.sep).join(path.posix.sep).includes(i.specFilePath)
```

Comment:
I see what you're doin here. This is a creative solution to always forcing linux style!